### PR TITLE
[Media] new test for this module

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request adds new test functions for this module.
todo rebase

### CODE REVIEW

**Comment by driusan** [comment]
@kongtiaowang has conflicts and needs to be merged

File: modules/media/test/MediaTest.php [hunk 1]
```diff
@@ -27,7 +27,26 @@
...
+            //Browse table header
+            'No.'         => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'File Name'   => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'PSCID'       => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Visit Label' => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Instrument'  => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Site'        => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Uploaded By' => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Date Taken'  => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
+            'Comments'    => '#dynamictable > thead:nth-child(1) > tr:nth-child(1)',
```
Comment (davidblader) [inline modules/media/test/MediaTest.php]: should Language be added here as well?

Comment (ridz1208) [reply modules/media/test/MediaTest.php]: yes

**Comment by ridz1208** [comment]
@kongtiaowang what is the problem here ? can it be fixed ?

**Comment by kongtiaowang** [comment]
it depends on #4145

**Comment by zaliqarosli** [comment]
@driusan @kongtiaowang this merge is breaking travis on my PRs

**Pull request merged**